src/game/client.cpp

```diff
@@ -496,15 +496,14 @@ namespace client
         vector<int> items;
         if(list && *list)
         {
-            vector<char *> chunk;
+            std::vector<std::string> chunk;
             explodelist(list, chunk);
             loopv(chunk)
             {
-                if(!chunk[i] || !*chunk[i] || !isnumeric(*chunk[i])) continue;
-                int v = parseint(chunk[i]);
-                items.add(v ? 1 : 0);
+                if(chunk[i].empty() || !isnumeric(chunk[i][0])) continue;
+                int v = parseint(chunk[i].c_str());
+                items.emplace_back(v >= W_OFFSET && v < W_ITEM ? v : 0);
```

Comment:
This comparison has been changed significantly. Will have to check the values of `W_OFFSET` and `W_ITEM` before approving this PR.